banners/desktop/components/BannerVar.vue

```diff
@@ -42,23 +42,38 @@
 			</template>
 
 			<template #footer>
-				<BannerFooter @showFundsModal="isFundsModalVisible = true" />
+				<FooterAlreadyDonated
+					@showFundsModal="isFundsModalVisible = true"
+					@showAlreadyDonatedModal="isAlreadyDonatedModalVisible = true"
+				/>
 			</template>
 
 		</MainBanner>
 
 		<SoftClose
 			v-if="contentState === ContentStates.SoftClosing"
 			@close="() => onClose( 'SoftClose', CloseChoices.Close )"
-			@maybe-later="() => onClose( 'SoftClose', CloseChoices.MaybeLater )"
-			@time-out-close="() => onClose( 'SoftClose', CloseChoices.TimeOut )"
+			@maybeLater="() => onClose( 'SoftClose', CloseChoices.MaybeLater )"
+			@timeOutClose="() => onClose( 'SoftClose', CloseChoices.TimeOut )"
 		/>
 
 		<FundsModal
 			:content="useOfFundsContent"
 			:is-funds-modal-visible="isFundsModalVisible"
 			@hideFundsModal="isFundsModalVisible = false"
 		/>
+
+		<AlreadyDonatedModal
+			:is-visible="isAlreadyDonatedModalVisible"
+			:is-already-donated-modal-visible="isAlreadyDonatedModalVisible"
```

Comment:
as far as I can see only `:is-visible` is used as a prop in the component, right?
`:is-already-donated-modal-visible="isAlreadyDonatedModalVisible"` can probably be removed